docs/essentials/launcher.md

```diff
@@ -35,6 +35,27 @@ public class LauncherTest
 }
 ```
 
+## Platform Differences
+
+# [Android](#tab/android)
+
+Calling the `CanOpenAsync` method does not add any delay as it returns immediately. The `Result` of that Task is available immediately after the call.
```

Comment:
I don't like putting Result in there because that sort of hits that devs should call .Result which should never be done. I would leave this off of both.

Reply:
So ... I don't understand. I agree with James that it can confuse developers on what they are supposed to do with that result, but in the same time it's important for them to know that this async method doesn't add a delay of any sort... 
Maybe I'll leave only the first sentence then.

Reply:
Close. As I commented above, the first sentence isn't quite what you're trying to get across. It's not a platform-specific feature that `CanOpenAsync` *returns* immediately; all Async methods do that. The question devs want answered is when does the task returned by `CanOpenAsync` *complete*. Also, since we you can guarantee that the task always completes successfully (never faulted or canceled), you can state that too. So something like this:
> The `Task` returned from `CanOpenAsync` always completes immediately and always completes successfully.